src/factories/nodeStopFactory.ts

```diff
@@ -1,6 +1,8 @@
 import INodeStop from '~/models/INodeStop';
+import IExternalNodeStop  from '~/models/externals/IExternalNodeStop.ts';
+
 export default class StopFactory {
-    public static createStop = (node: any): INodeStop => {
+    public static createStop = (node: IExternalNodeStop): INodeStop => {
```

Comment:
Why is this named NodeStop and not just Stop?

Reply:
It is named after the models we're using. As for in this case, after "INodeStop". The changes you requested makes sense but need renaming of the models which shouldn't be done in this task.

Reply:
Add this as a new task then and it'll be fine.

Reply:
Outdated, i renamed it as IStop